invenio_cli/helpers/scripts.py

```diff
@@ -0,0 +1,298 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Previous script files helper methods."""
+
+import errno
+import logging
+import os
+import shutil
+import signal
+import subprocess
+import time
+from pathlib import Path
+
+import click
+from flask import current_app
+from flask.cli import with_appcontext
+from invenio_base.app import create_cli
+
+from .log import LogPipe
+
+
+def run_command(cli, runner, command, message=None, catch_exceptions=False,
+                verbose=False):
+    """Run [invenio] command."""
+    click.secho("{}...".format(message if message else command),
+                fg="green")
+    res = runner.invoke(cli, command, catch_exceptions=catch_exceptions)
+    if verbose:
+        click.secho(res.output)
+
+
+#############
+# BOOTSTRAP #
+#############
+
+
+def _bootstrap_dev(pre, loglevel, logfile, verbose):
```

Comment:
This seems to indicate that log semantics have been diluted and then need to be condensed back.

What if we capture these "log information" in a single data structure and pass *it* around? This information always goes together.

Reply:
Encapsulate logfile and verbose.

Open issue about loglevel and remove until we get to it.